src/main/groovy/de/dkfz/config/ResourceSet.groovy

```diff
@@ -0,0 +1,104 @@
+package de.dkfz.config
+
+import de.dkfz.eilslabs.tools.conversion.BufferValue
+import de.dkfz.eilslabs.tools.conversion.ConversionHelperMethods
+import de.dkfz.eilslabs.tools.conversion.TimeUnit
+
+
+public class ResourceSet {
+    private final String queue;
+    private ResourceSetSize size;
+    /**
+     * The target memory value.
+     */
+    private BufferValue mem;
+    private BufferValue memMax;
+
+    private Integer cores;
+    private Integer coresMax;
+    private Integer nodes;
+    private Integer nodesMax;
+    private TimeUnit walltime;
+
+    /**
+     * Hard disk storage used.
+     */
+    private BufferValue storage;
+    private BufferValue storageMax;
+    private String additionalNodeFlag;
```

Comment:
Somehow it is, somehow not. We'll have to see, how this will be of use in the future.